packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1250,6 +1250,31 @@ describe('build', () => {
       (await fs.readFile(join(output, 'static/index.txt'), 'utf8')).trim()
     ).toEqual('marketing');
   });
+
+  it('should write to flags.json', async () => {
+    if (process.platform === 'win32') {
+      console.log('Skipping test on Windows');
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (soft-blocking):** Why does this fail on Windows?

If we need to skip this test on Windows, can we add a comment about why?‏

Reply:
Sure, added a comment! Can't run it on windows because the fixture runs a bash script.

Reply:
Can we change the fixture to not use a bash script? I'd like to keep windows coverage if we can.

Reply:
Fair, converted it to a node script.